src/main/plugins/org.dita.pdf2/cfg/fo/i18n/ko.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- 
+Copyright © 2004-2006 by Idiom Technologies, Inc. All rights reserved. 
```

Comment:
This isn't Idiom code, thus remove Idiom copyright and just use the normal OT statement.